__tests__/endpoints.test.js

```diff
@@ -10,13 +10,13 @@ const {
 const endpoints = require("../endpoints.json");
 const app = require("../app");
 
-beforeEach(() => {
-  return seed({ articleData, commentData, topicData, userData });
-});
+// beforeEach(() => {
+//   return seed({ articleData, commentData, topicData, userData });
+// });
 
-afterAll(() => {
-  return db.end();
-});
+// afterAll(() => {
+//   return db.end();
+// });
```

Comment:
Commented out code should not usually appear in production ready code. Additionally these should not be commented out as they serve a very important job!

Reply:
sorry! i thought they were tripping up a test, but it wasn't them! will change